### PR TITLE
RHDEVDOCS-4541: Adding default prunner configuration

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.11`, `enterprise-4.12`
- **JIRA issues**: [RHDEVDOCS-4541 Add default prunner configuration](https://issues.redhat.com/browse/RHDEVDOCS-4541)
- **Preview pages**: [Default prunner configuration](http://file.pnq.redhat.com/sosarkar/4541-add-default-prunner-settings/cicd/pipelines/automatic-pruning-taskrun-pipelinerun.html#default-prunner-configuration_automatic-pruning-taskrun-pipelinerun)
- **SME review**: @pradeepitm12  
- **QE review**: @ppitonak / @VeereshAradhya

### CODE REVIEW

File: modules/op-default-prunner-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+...
+pruner:
+  keep: 100
+  resources:
+  - pipelinerun
+  schedule: 0 8 * * *
+...
+----
+
+You can override the default prunner configuration by using annotations.
```
Comment (pradeepitm12) [inline modules/op-default-prunner-configuration.adoc:22]: How about this.
You can override the default prunner configuration for a namespace by using annotations on namespace.

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

File: modules/op-default-prunner-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// cicd/pipelines/automatic-pruning-taskrun-pipelinerun.adoc
+
+:_content-type: REFERENCE
+[id="default-prunner-configuration_{context}"]
+= Default prunner configuration
+
+The default configuration for periodic prunning of pipeline runs is as follows:
```
Comment (VeereshAradhya) [inline modules/op-default-prunner-configuration.adoc:9]: `pruning`?

**Review by ppitonak — APPROVED** [review]

**Review by gabriel-rh — COMMENTED** [review]
Looks good - some minor sugestions

File: modules/op-default-prunner-configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+//
+// cicd/pipelines/automatic-pruning-taskrun-pipelinerun.adoc
+
+:_content-type: REFERENCE
+[id="default-prunner-configuration_{context}"]
+= Default prunner configuration
+
+The default configuration for periodic pruning of resources associated with pipeline runs is as follows:
+
+[source,yaml]
```
[the remark this answers is not in the document]
Comment (sounix000) [reply modules/op-default-prunner-configuration.adoc:11]: @pradeepitm12 / @VeereshAradhya : Please help me with the information asked above by Gabriel.

[the remark this answers is not in the document]
Comment (VeereshAradhya) [reply modules/op-default-prunner-configuration.adoc]:
```
apiVersion: operator.tekton.dev/v1alpha1
kind: TektonConfig
metadata:
  name: config
...
spec:
  pruner:
    keep: 100
    resources:
    - pipelinerun
    schedule: 0 8 * * *
...
```

[the remark this answers is not in the document]
Comment (sounix000) [reply modules/op-default-prunner-configuration.adoc]: Thanks Veeresh!

**Review by gabriel-rh — APPROVED** [review]
/lgtm

**Pull request merged**